_sass/scaffolds.scss

```diff
@@ -255,6 +255,9 @@ li.sticky-nav{
 .project-banner-wrapper {
   display: flex;
   justify-content: space-between;
+  border-bottom: 3px;
+  border-style: none none dashed none;
```

Comment:
hmm, i can't say that i'm very compelled by the dashed border 😕 

...hoping this comes across in the nicest way possible. from what i gathered in the meeting with Yelena is that the 2-column layout of the "emergency INDEX AN ANNUAL DOCUMENT OF PERFORMANCE PRACTICE" & rand0 image would convert to ...something else? from the doc "Maybe under the first header on each age it appears?". i'm not sure that the rand0 image on it's own line/row makes much sense and will leave a lot of white space.

Reply:
going to hit this on the future branch - i don't like the dash either - and i definitely agree that the rand0 image on its own is no good...